src/commonMain/kotlin/spaceEngineers/controller/CharacterControllerExtensions.kt

```diff
@@ -50,6 +50,22 @@ fun Character.blockingMoveForwardByDistance(
     error("timeout after $maxTries tries")
 }
 
+fun Character.blockingMoveBackwardsByDistance(
+    distance: Float,
+    velocity: Float = 1f,
+    maxTries: Int = 1000,
+    startPosition: Vec3,
+): SeObservation {
+    repeat(maxTries) {
+        moveForward(velocity = -velocity).let {
```

Comment:
Why not just call the forward version with negative velocity? is this function adding anything else then just the comfort of not using negative velocity?

Reply:
I could have swore, there was a reason for that and that there was another difference in the method, but looks like there isn't :-D well spotted